src/deepsparse/v2/text_generation/pipeline.py

```diff
@@ -140,12 +154,22 @@ def __init__(
             "multi_engine": "compile_logits",
             "compile_logits": [
                 "multi_engine_prefill",
+                "prep_for_generation",
```

Comment:
love how readable the routes are, I wonder if we can plug this into a graph-viz tool to get an automatic view of the DAG

Reply:
Sick idea.